packages/glasscoder/PKGBUILD

```
@@ -0,0 +1,29 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: this.ven <misgi@ven.uber.space>
+
+pkgname=glasscoder
+pkgver=1.0.4
+pkgrel=1
+pkgdesc="Minimalist audio encoder for IceCast and Shoutcast streaming
+servers as well as HTTP Live Streaming (HLS)"
+arch=(x86_64)
```

```suggestion
arch=(x86_64 aarch64)
```

Since fd11c14296449542c90300129c46b223881d5003 the CI doesn't build aarch64 automatically, so I'd recommend to first always add it explicitly and if the build fails, try to fix or just remove it again 🤔

---

It works. :smiley: